weaviate/collections/classes/types.py

```diff
@@ -32,6 +66,7 @@ def _to_dict(self) -> Dict[str, float]:
     datetime.datetime,  # date
     uuid_package.UUID,  # uuid
     GeoCoordinate,  # geoCoordinates
+    Union[PhoneNumber, PhoneNumberType],  # phoneNumber
```

Comment:
why add both? I thought one is input and one is output?

Reply:
So this is an issue of having the `Properties` generic apply for both input and output types. Until now, this was true: inputs and outputs look the same. But for phone numbers, they are not the same due to the backend parsing of the input string

In order to maintain the typing convention, `PhoneNumber` must be in their for when users supply generics within `.data` and `PhoneNumberType` must be in their for when users access generics within `.query(generate)`

This is a small limitation of Python's typing system as I know of a solution for this in TS